pkg/network/vm/controller.go

```diff
@@ -16,7 +16,7 @@
  *
  */
 
-package network
+package vm
```

Comment:
Please share what this `network.vm` package represents now.
It is pretty generic, covering all "vm" network things?

To be clear, in the prev location, it represented the "network" logic of the VM & VMI controllers.
Now that it is moved to the "network" tree, it needs to reflect some other hierarchy.

BTW, we also have the `vmicontroller` package. How does it fit in the structure?

Reply:
I've renamed `vmicontroller` to `controllers` and moved VM controller under it.